src/modules/project/pages/projectView/views/projectActivityPanel/screens/fundingForm/views/FundingFormSection.tsx

```diff
@@ -14,14 +14,21 @@ type Props = {
 
 export const FundingFormSection = ({ onBackClick }: Props) => {
   const { t } = useTranslation()
+
+  const { setProjectGoalId } = useProjectContext()
   const {
     fundForm: { setState },
   } = useFundingContext()
 
+  const handleOnBackClick = (event: MouseEvent<HTMLButtonElement>) => {
+    setProjectGoalId(null)
```

Comment:
if the projectGoalId is null, we will refer the default current goal right?

Reply:
Yes exactly, the logic was implemented in the backend so that the default current goal is selected if no projectGoalId is provided